content/integrations/dropwizard/_index.md

```diff
@@ -6,15 +6,15 @@ tags: ["#dropwizard", "#integrations", "#custom metrics"]
 author: Lawrence Lane
 ---
 
-Dropwizard is part Java framework and part Java library that assists in operating web services. Dropwizard will take your web application and run it locally, recording metrics on its performance. You can integrate with Dropwizard via our custom Dropwizard Metrics Library to send these metrics to a StatsD server, which you can then forward to Metricly.
+Dropwizard is part Java framework and part Java library that assists in operating web services. Dropwizard will take your web application and run it locally, recording metrics on its performance. You can integrate with Dropwizard via our custom Dropwizard Metrics Library to send these metrics to a StatsD server, which you can then forward to CloudWisdom.
 
 ## Configuration
 1. Include the appropriate Ananke library dependency. You’ll also need a working StatsD (Metricly StatsD or Etsy StatsD) integration.
 2. Include the appropriate dropwizard-metrics dependency for the build manager of your choice.
 
 ```
 <dependency>
-  <groupId>com.metriclyicly</groupId>
```

Comment:
It's still netuitive: https://github.com/Netuitive/dropwizard-metrics#using-the-dropwizard-integration